packages/devtools_app/lib/src/network/flutter/http_headers_tab.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:recase/recase.dart';
```

Comment:
What is this package for?

Reply:
I'm using this package to convert the keys in the timeline events from `lowerCamelCase` to `Sentence Case` so they can be displayed more cleanly. I'm sure this is something we could implement ourselves if we don't want to depend on a third-party package, but this was more a time saver on my end.